api/controllers/auth_controller.go

```diff
@@ -2,14 +2,14 @@ package controllers
 
 import (
 	"net/http"
-	"strings"
 
-	"github.com/PTSS-Support/identity-service/api/dtos/requests"
+	requests "github.com/PTSS-Support/identity-service/api/dtos/requests/auth"
```

Comment:
tested this in code, but why have requests here requests here at the beginning whilst all other classes don't use it?
Inconsistent smh
but my ide gave no errors when removing it, so why have it?

Reply:
nvm found out that you have it quite a lot in this code. Is this necessary though?

Reply:
Idk why it doesn't work. My suspicion is that it has something to do with the folder path (e.g. namespace does not path last folder naming) but idk.

Reply:
hmm weird, alright, lets keep it then